zerver/migrations/0389_message_edit_history_visibility.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.2.12 on 2022-04-15 19:53
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("zerver", "0388_preregistrationuser_created_user"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="realm",
+            name="allow_edit_history",
+        ),
+        migrations.AddField(
+            model_name="realm",
+            name="message_edit_history_visibility",
+            field=models.PositiveSmallIntegerField(default=1),
+        ),
+    ]
```

Comment:
This migration doesn't transfer the existing settings. See https://zulip.readthedocs.io/en/latest/subsystems/schema-migrations.html